src/components/OrgUpcomingEvents.tsx

```diff
@@ -0,0 +1,39 @@
+import { api } from "@src/utils/api";
+import { type FC } from "react";
+import { SelectClub as Club } from "@src/server/db/models";
+
+const cur_time = new Date();
```

Comment:
Move this inside the component. Only the code inside the component will be executed again when the page is refreshed.

Reply:
Putting the date inside the component causes the date to be refresh too quickly that useQuery does not pull for the event api. This is the console.log(cur_event). It refreshes really quickly.
![image](https://github.com/UTDNebula/jupiter/assets/90099089/93c37c85-aa00-44ff-ad96-517baa2d936e)

I thought about using useEffect and useState to set the interval to refresh and pull from the API every 60 seconds, but the issue is every minute, the event is reloaded which means the events disappear and reappear again. To fix this, I useState() and set that time whenever the user loads the components. Let me know if there is a better solution than this approach.